fdbrpc/FlowTransport.actor.cpp

```diff
@@ -633,7 +636,35 @@ TransportData::~TransportData() {
 	}
 }
 
+BYZANTINE(wrongEndpoint)
+
 ACTOR static void deliver(TransportData* self, Endpoint destination, ArenaReader reader, bool inReadSocket) {
+	if (BYZANTIFY(wrongEndpoint)) {
+		TEST(true) // Injected wrongEndpoint
+		std::poisson_distribution<unsigned> dist(5);
+		uint64_t first = dist(*deterministicRandom()), second = dist(*deterministicRandom());
```

Comment:
What is the chance that the input `destination` is transformed to another  valid destination?

My understanding is
If the alternated destination is not a valid destination, it is equivalent to "broken network" in the current simulation. 
So the byzantine error is to deliver a message that should arrive at dest1 to dest2. Is that correct?

Reply:
> So the byzantine error is to deliver a message that should arrive at dest1 to dest2
That's correct

Reply:
I think this is a bad byzantine injection as we already test for these kind of scenarios... I mostly have this here (for now) so that I can test that this actually happens in a test I am going to write.